packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -13,19 +21,17 @@ use arrow::{
     ipc::writer::{IpcDataGenerator, IpcWriteOptions},
     util::bit_util,
 };
-use futures::FutureExt;
-use mv8::MiniV8;
+use futures::{Future, FutureExt};
 use tokio::{
     sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender},
     task::JoinError,
 };
 use tracing::{Instrument, Span};
 
 pub use self::error::{Error, Result};
-use self::mini_v8 as mv8;
 use super::comms::{
     inbound::InboundToRunnerMsgPayload,
-    outbound::{OutboundFromRunnerMsg, OutboundFromRunnerMsgPayload},
+    outbound::{OutboundFromRunnerMsg, PackageError, UserError, UserWarning},
     ExperimentInitRunnerMsg, MessageTarget, NewSimulationRun, RunnerTaskMsg, TargetedRunnerTaskMsg,
```

Comment:
As done in #431, could you make the import absolute, please? 
https://github.com/hashintel/hash/blob/33cef632a01f6c6cc12aeeb8c3b45396a0e8dbd3/packages/engine/src/worker/runner/javascript/mod.rs#L48-L56

Reply:
Imports are now absolute https://github.com/hashintel/hash/pull/414/commits/ec3517a4bfc25639e47e72d6bad71fceb40e405d